### ISSUE TITLE
Point to OmniLayer/omnicore

### ISSUE THREAD

**Issue by dexX7** [mastercoin-MSC/mastercore#306]
There should be a very visible reference of the new repo location:

https://github.com/OmniLayer/omnicore

**Comment by zathras-crypto** [comment]
Any suggestions on what to use for this?

**Comment by dexX7** [comment]
A readme.md file with a huge "moved to xyz" note. It's not that straight forward though, because until now the "main" branch was the "last stable" branch, and adding a note to the 0.0.10 branch has probably no visibility.

So how about.. creating a new, empty branch, which has only a readme with the note, as well as links to the 0.0.8 and 0.0.9 branches? This branch could then be used as "main" branch, and should probably called "master".

As far as I can see this should not interfere with the CI server, which explicitly uses the `mscore-0.0.9` branch for the `omnicore-integration` job, but final confirmation of @msgilligan wouldn't hurt nevertheless.

**Comment by dexX7** [comment]
Somewhat related:

http://www.omnilayer.org/ points to [mastercoin-MSC/mastercore](https://github.com/mastercoin-MSC/mastercore), which is probably fine at the very moment, but more importantly: there is a reference of [mastercoin-MSC/mastercore/blob/mscore-0.0.8/doc/build-unix.md](https://github.com/mastercoin-MSC/mastercore/blob/mscore-0.0.8/doc/build-unix.md), which is outdated, and it should probably point to the latest release instead.

**Comment by msgilligan** [comment]
Yes, that should be fine as long as mscore-0.0.9 is still there. At some point, I'll need to point to the new repo and `omnicore-integration` should come from 0.0.10 and `omnicore-stable` should be 0.0.9. This will be complicated by the fact that they're going to want different blockchains.

**Comment by dexX7** [comment]
Bump.

**Comment by dexX7** [comment]
Bump. @zathras-crypto @CraigSellars @achamely

**Comment by zathras-crypto** [comment]
> So how about.. creating a new, empty branch, which has only a readme with the note, as well as links to the 0.0.8 and 0.0.9 branches?

I don't think we should in any way be encouraging people to use 0.0.8 branches, they're out of consensus and should not be used under any circumstances I can think of (no STO support).  Other than that I'm happy to use this method and just stick a readme in there to point to the old repo.

> This branch could then be used as "main" branch, and should probably called "master".

What would you see happen when we release 0.0.10?  Amend the readme to point to that branch or publish 0.0.10 into master?

> Somewhat related:  http://www.omnilayer.org/ 

Can't help there I'm afraid, I have no access or info on that site.

**Comment by dexX7** [comment]
> I don't think we should in any way be encouraging people to use 0.0.8 branches ...

Good point! I was just thinking about pointing out "look, the old code isn't gone, it's still there".

> I'm happy to use this method and just stick a readme in there to point to the old repo.
> What would you see happen when we release 0.0.10?

Ah, I was actually thinking about a `README.md` for `mastercoin-MSC/mastercore`, to reference the new location `OmniLayer/omnicore`, so users can learn about the move.

**Comment by zathras-crypto** [comment]
> so users can learn about the move

That makes much more sense :)  Though I'm not sure we should be pointing people here yet or they may clone and build omnicore-0.0.10 deploying it before it's released?

**Comment by dexX7** [comment]
I see not much harm, if users start to _test_ the 0.10 based branch, but using it in production environments is probably less ideal, even though it shouldn't be consensus breaking. How about something like:

---

This project has moved to:

https://github.com/OmniLayer/omnicore

Note: `omnicore-0.0.10` is not ready, nor intended, for mainnet use at this point.

The **latest stable release** `0.0.9.1-rel` can be found here:

https://github.com/mastercoin-MSC/mastercore/releases/tag/v0.0.9.1

To keep up with the current progress, please keep an eye on the the Omni blog:

http://blog.omni.foundation

---

**Comment by zathras-crypto** [comment]
> I see not much harm, if users start to test the 0.10 based branch

Absolutely, the more testing the better :)

> How about something like

That sounds good to me - I think I have the rights needed to do it...   Before I do any other feedback anyone?

**Comment by dexX7** [comment]
Who maintains http://www.omnilayer.org/?

It's a no-go, in my opinion, to point to mastercore-0.0.8, which is out of consensus, so I'd really like to see an update on this.

Further, OmniJ should be listed there as well.

**Comment by zathras-crypto** [comment]
Paging @achamely @CraigSellars - they have control over the website AFAIK.

**Comment by achamely** [comment]
Pages in github sometimes seem to get lost in my email (trying to update my filters to be more recognizing of them.) I've updated omnilayer.org for the moment to at least point to the 0.0.9 branch/releases/api etc..   @zathras-crypto if we want to get an announcement.rd file or add a new section to the Readme.Rd file on the 0.0.9 branch i can directly point to that for the udpate for now. 

@dexX7's wording looks pretty spot on to me.
